hs_collection_resource/templates/pages/collectionresource.html

```diff
@@ -0,0 +1,553 @@
+                                        {% if res|user_permission:request.user.pk|lower == "none" %}
+                                            <strong class="label-private">{{ res|user_permission:request.user.pk }}</strong>
+                                        {% elif res|user_permission:request.user.pk|lower == "open access" %}
+                                            <strong class="label-public">{{ res|user_permission:request.user.pk }}</strong>
+                                        {% else %}
+                                            {{ res|user_permission:request.user.pk }}
+                                        {% endif %}
+                                    </td>
+                                    <td>
+                                        <span onclick="remove_collection_item('{{ res.short_id }}')" data-form-id="form-favorite-{{ res.short_id }}" data-form-type="toggle-favorite"
+                                                          class="glyphicon glyphicon-remove btn-inline-favorite"></span>
+                                    </td>
+                                </tr>
+                            {% endfor %}
+                         {% endif %}
+                        </tbody>
+                    </table>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-default" data-dismiss="modal" id="save-collection-btn-cancel">
+                       Cancel
+                    </button>
+                    <button type="button" class="btn btn-success" data-dismiss="modal" id="save-collection-btn-ok" onclick="add_collection_item();">
+                        Add to collection
+                    </button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+    {#    modal div: deleted-resources-modal #}
+    <div class="modal fade" id="deleted-resources-modal" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog" style="width:65%">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+                    <h4 class="modal-title" id="save-collection-warning-title">Deleted Resources</h4>
+                </div>
+                <div class="modal-body" id="save-collection-warning-body">
+                    <table id="deleted-resources-table" class="table-hover table-striped" width="100%">
+                        <thead>
+                        <tr>
+                            <th>Resource ID</th>
+                            <th>Title</th>
+                            <th>Type</th>
+                            <th>Deleted by</th>
+                            <th>Date</th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        {% if deleted_resources %}
```

Comment:
@zhiyuli Again, this {% if... %} check is not needed. You can double check to make sure this is indeed true.